### PR TITLE
Minor Editor/Config level fixes

### PR DESCRIPTION
**Pull request by Troy-Ferrell**
## Overview
This changes fixes some minor configurations and editor behavior issues.

- Fix isCustomProfile to false for DefaultSpatialAwarenessSystemProfile
- Fix check for SpatialAwareness layer 31
- Fix HelpURL and documentation inspector button for pointers

## Changes
- Fixes: #5679 , #5680 , #4648 

## Verification
All tests passed

> As a reviewer, it is possible to check out this change locally by using the following
> commands (substituting {PR_ID} with the ID of this pull request):
>
> git fetch origin pull/{PR_ID}/head:name_of_local_branch
>
> git checkout name_of_local_branch

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealitySpatialAwarenessSystemProfile.asset [hunk 1]
```diff
@@ -12,7 +12,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 19f279aded72cb741b4de89a54359dd4, type: 3}
   m_Name: DefaultMixedRealitySpatialAwarenessSystemProfile
   m_EditorClassIdentifier: 
-  isCustomProfile: 1
+  isCustomProfile: 0
```
Comment (wiwei) [inline Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealitySpatialAwarenessSystemProfile.asset]: Note to self - I can update the presubmit script to ensure this doesn't happen again (i.e. check that none of the checked in profiles has isCustomProfile: 1

File: Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs [hunk 2]
```diff
@@ -296,6 +310,33 @@ private static void CheckOptimalConfiguration()
...
+        /// <param name="layerNumber">index of layer to modify. Should be between 8 and 31</param>
+        /// <param name="layerName">Name of layer to set</param>
+        private static void CreateLayer(int layerNumber, string layerName)
+        {
+            SerializedObject tagManager = new SerializedObject(AssetDatabase.LoadAllAssetsAtPath("ProjectSettings/TagManager.asset")[0]);
+
+            SerializedProperty layers = tagManager.FindProperty("layers");
+            if (layers == null || !layers.isArray)
+            {
+                Debug.LogWarning("Can't set up the layers.  It's possible the format of the layers and tags data has changed in this version of Unity.");
```
Comment (wiwei) [inline Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs]: Nit: Consider adding a comment or something around here suggesting the user manually add the layer themselves with the given name + ID. At least that way they'd have a workaround if they hit this warning.

File: Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs [hunk 3]
```diff
@@ -87,38 +89,46 @@ private static void ShowSettingsDialog()
...
-                message += "\nWould you like to make this change?";
+                // Only make change if not already set. Regardless of whether it is already SpatialAwareness or something user set
+                var isSpatialLayerAvailable = string.IsNullOrEmpty(LayerMask.LayerToName(SpatialAwarenessDefaultLayer));
+                if (isSpatialLayerAvailable)
+                {
+                    builder.AppendLine("- Set Default Spatial Awareness Layer");
+                }
 
-                if (!forceTextSerialization || !visibleMetaFiles || !PlayerSettings.virtualRealitySupported || !usingSinglePassInstancing)
+                builder.Append("\nWould you like to make this change?");
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs]:
```suggestion
                builder.Append("\nWould you like to make these changes?");
```

File: Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs [hunk 4]
```diff
@@ -296,6 +310,32 @@ private static void CheckOptimalConfiguration()
             }
         }
 
+        
+        /// <summary>
+        /// Helper method to update the a layer name at given index in the related project asset file
+        /// </summary>
+        /// <param name="layerNumber">index of layer to modify. Should be between 8 and 31</param>
+        /// <param name="layerName">Name of layer to set</param>
+        private static void CreateLayer(int layerNumber, string layerName)
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs]: We should likely remove this functionality from MixedRealityToolkit (i believe that is were it currently resides)

Comment (Troy-Ferrell) [reply Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs]: @davidkline-ms oh nice! didn't even know they already existed. I delete redundant code

**Pull request merged**